client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,57 @@
+import useStyles from './useStyles';
+import { Box, Button, Typography, Select, MenuItem, OutlinedInput, Grid } from '@material-ui/core';
+import { useEffect } from 'react';
+import PinIcon from './PinIcon.png';
+import fetchProfile from '../../helpers/APICalls/fetchProfile';
+import BookingForm from './BookingForm/BookingForm';
+
+const ProfileDetails = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.profileDetailsPage}>
+      <Box className={classes.profileDetails}>
+        <img
+          className={classes.backgroundImg}
+          src="https://assets-us-01.kc-usercontent.com/28e7bd12-5b30-009d-524e-785407f8bd6e/1105c802-3234-415b-a429-0a50c3259589/Stock%20WB%20Muirfield%20VIII.jpg?w=1600&h=900&fit=crop"
+          alt="user's background image"
+        />
+        <img
+          className={classes.userImg}
+          src="https://images.pexels.com/photos/415829/pexels-photo-415829.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500"
+          alt="user's profile image"
+        />
+        <Typography className={classes.userName}>Norma Byers</Typography>
+        <Typography className={classes.userTitle}>Loving pet sitter</Typography>
+        <Box className={classes.userLocationContainer}>
+          <img src={PinIcon} className={classes.userLocationIcon} alt="location pin icon" />
+          <Typography className={classes.userLocation}>Toronto, Ontario</Typography>
+        </Box>
+        <Box className={classes.userDescriptionContainer}>
+          <Typography className={classes.userDescriptionContainerHeader}>About me</Typography>
+          <Typography className={classes.userDescription}>
+            Animals are my passion! I will look after your pets with loving care. I have some availability for pet care
+            in my home as well. I have 10 yrs experience at the Animal Hospital, and have owned multiple pets for many
+            years, including numerous rescues. Kindly email, text or call me and I will respond promptly!
+          </Typography>
+        </Box>
```

Comment:
This portion of the UI fits the use-case of the [`Card` component](https://v4.mui.com/components/cards/#card) perfectly:
>Cards contain content and actions about a single subject.
 
Semantically, it would be so much clearer. For example, we can use the [`CardMedia` subcomponent](https://v4.mui.com/api/card-media/#cardmedia-api) for the cover and the profile image. We can then separate the header and the content, wrapping each respectively in [`CardHeader`](https://v4.mui.com/api/card-header/) and [`CardContent`](https://v4.mui.com/api/card-content/) subcomponents. Current version doesn't require any, but if we ever decide to add actions such as `bookmark` and/or `like` in the future, we can implement them seemlesly with the help of the [`CardActions`](https://v4.mui.com/api/card-actions/) subcomponent. There are some inspiring examples in the [docs](https://v4.mui.com/components/cards/#card), I'd really recommend taking a look! 🎉